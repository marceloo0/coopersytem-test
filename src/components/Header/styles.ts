import styled from 'styled-components/native';

export const Container = styled.View`
  background-color: blue;
  height: 60px;
  align-items: center;
  justify-content: center;
  border-bottom-color: #ffff00;
  border-bottom-width: 5px;
`;
export const Title = styled.Text`
  color: #fff;
  font-size: 21px;
  font-weight: bold;
`;
