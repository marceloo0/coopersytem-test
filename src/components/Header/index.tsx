import React from 'react';

import {Container, Title} from './styles';

const Header: React.FC = () => {
  return (
    <Container>
      <Title>Resgate</Title>
    </Container>
  );
};

export default Header;
