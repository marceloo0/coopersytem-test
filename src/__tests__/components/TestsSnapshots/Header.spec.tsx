import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../../components/Header';

test('renders correctly', () => {
  const header = renderer.create(<Header />).toJSON();
  expect(header).toMatchSnapshot();
});
