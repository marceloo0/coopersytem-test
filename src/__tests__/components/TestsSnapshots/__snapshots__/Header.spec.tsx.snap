// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "blue",
        "borderBottomColor": "#ffff00",
        "borderBottomWidth": 5,
        "height": 60,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontSize": 21,
          "fontWeight": "bold",
        },
      ]
    }
  >
    Resgate
  </Text>
</View>
`;
